src/App.js

```diff
@@ -1,13 +1,30 @@
 import React from "react";
 
 import Bookings from "./Bookings";
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristInfoCards";
+import Footer from "./Footer";
 import "./App.css";
+import Cities from "./data/cities.json";
+import Restaurant from "./Restaurant";
 
 const App = () => {
+  let footerArray = [
+    "123 Fake Street, London, E1 4UD",
+    "hello@fakehotel.com",
+    "0123 456789",
+  ];
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
       <Bookings />
+      <div className="info-cards">
+        {Cities.map((city) => {
+          return <TouristInfoCards key={city.id} city={city} />;
```

Comment:
I would say that this would make sense to be called `TouristInfoCard` over `TouristInfoCards` as it is just one card. If we did the map inside the card i would call it what you did ☺️

Reply:
Yes, it's true:) I've changed it:) Thank you!